tests/features/api.feature

```diff
@@ -0,0 +1,283 @@
+      And the "oe_author[0][target_id]" field should contain "Directorate-General for Communication (http://publications.europa.eu/resource/authority/corporate-body/COMMU)"
+      And the "oe_content_content_owner[0][target_id]" field should contain "Directorate-General for Communication (http://publications.europa.eu/resource/authority/corporate-body/COMMU)"
+
+      # Add related entities, such as venues and contacts and reload the page.
+      When the following Default Venue entity:
+        | Name     | DIGIT                                                                                      |
+        | Address  | country_code: BE - locality: Brussels - address_line1: Rue Belliard 28 - postal_code: 1000 |
+        | Capacity | 12 people                                                                                  |
+        | Room     | B-28 03/A150                                                                               |
+      And the following Press Contact entity:
+        | Name               | A press contact                                                                          |
+        | Address            | country_code: HU - locality: Szeged - address_line1: Press contact 1 - postal_code: 6700 |
+        | Email              | press@example.com                                                                        |
+        | Phone number       | +32477777778                                                                             |
+        | Social media links | uri: http://facebook.com - title: Facebook - link_type: facebook                         |
+      And the following General Contact entity:
+        | Name               | A general contact                                                                            |
+        | Address            | country_code: HU - locality: Budapest - address_line1: General contact 1 - postal_code: 1011 |
+        | Email              | general@example.com                                                                          |
+        | Phone number       | +32477792933                                                                                 |
+        | Social media links | uri: http://instagram.com - title: Instagram - link_type: instagram                          |
+      And the Event Content "My event" is updated as follows:
+        | Venue                 | DIGIT                               |
+        | Contact               | A press contact, A general contact  |
+        | Organiser is internal | No                                  |
+        | Organiser name        | Name of the organiser               |
+      And I reload the page
+      And the "Organiser name" field should contain "Name of the organiser"
+
+      And I press "Edit" in the "A general contact" row
+      And I wait for AJAX to finish
+      And the "Phone number" field should contain "+32477792933"
+      And the "Country" field should contain "HU"
```

Comment:
we should verify the rest of the address and the social media,
also check values of venue and press contact

Reply:
I added statements which verify the values in the entity's View page.